frontend/src/v5/ui/components/shared/revisionDetails/revisionDetails.component.tsx

```diff
@@ -58,7 +61,19 @@ export const RevisionDetails = ({ containerId, revisionsCount = 1 }: IRevisionDe
 		}
 	}, []);
 
-	if (!isLoading && revisions && revisions.length === 0) {
+	if (revisionsCount === 0) {
+		if (isBusyInBackend(status)) {
```

Comment:
In other components we used a lot the notation
```jsx
{isBusyInBackend(status) && ( 
   <Component>
       ...
   </Component>
}
```

Maybe we can use that here too to be more consistent?